dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -465,13 +491,12 @@ LANStatusUiDetailsDlg(
 
                     leaseExpire = localtime(&pCurAdapter->LeaseExpires);
 
-                    if (wcsftime(szBuffer, 100, L"%m/%d/%Y %H:%M:%S %p", leaseExpire) != 0)
+                    if (tmToStr(leaseExpire, &leaseST, szBuffer))
                         SendMessageW(hDlgCtrl, LVM_SETITEMW, 0, (LPARAM)&li);
                 }
             }
 
             dwSize = 0;
-            li.iItem = InsertItemToListView(hDlgCtrl, IDS_DNS_SERVERS);
```

Comment:
@EricKohl, could you double-check? As you recently (fc882c48de225ec3e83f31ad25bda3331b58bf16) added this line.

Reply:
No, he did not add this line... he added the second occurence, which is indeed needed but did not remove this one, which becomes useless and leads to an empty line. @EricKohl : correct me if i'm wrong :)

Reply:
Did he not?
`+            li.iItem = InsertItemToListView(hDlgCtrl, IDS_DNS_SERVERS);`

Reply:
Right. But indeed I'd say only one is needed. To be discussed with Eric.